server/homebrew.api.js

```diff
@@ -187,7 +189,13 @@ const newBrew = async (req, res)=>{
 
 const updateBrew = async (req, res)=>{
 	// Initialize brew from request and body, destructure query params, set a constant for the google id, and set the initial value for the after-save method
-	let brew = _.assign(req.brew, excludePropsFromUpdate(req.body));
+	const updateBrew = excludePropsFromUpdate(req.body);
```

Comment:
Let's be careful about using the same variable name as the function here. I think earlier we had an issue of "X is not a function" because of variable name conflicts.

Reply:
OOP I did not noticed I'd done that. Changing that now